modules/managed-identity/user-assigned-identity/MOVED-TO-AVM.md

```diff
@@ -0,0 +1 @@
+This module has already been migrated to [AVM](https://github.com/Azure/bicep-registry-modules/tree/main/avm/res). Only the AVM version is expected to receive updates / new features. Please do not work on improving this module in [CARML](https://aka.ms/carml).
```

Comment:
Please also run the 'Set-Module' utility which will add this snippet to the Readme too 😊

Reply:
readme updated